pipdeptree.py

```diff
@@ -524,6 +524,7 @@ def render_json(tree, indent):
     :rtype: str
 
     """
+    tree = tree.sort()
```

Comment:
Any particular reason for this change?

Reply:
To make every run produces identical output (in Python2).

In Python2, the output changes (the order of keys is changed, thus, the hash will change) each time, which makes the output non-deterministic.

This is done in render_json_tree(), too.

Reply:
Ok. I run integration tests[1] to see if they need to be modified and then merge the PR this weekend. Sorry about the delay. 

[1] https://github.com/naiquevin/pipdeptree/blob/master/tests/e2e-tests